### DISCUSSION TITLE
[proxy server plugin] request rate limitation applied for remote 127.0.0.1

### DISCUSSION THREAD

**Discussion by rusmongoose** [glpi-project/glpi-agent#415] [Q&A]
### Documentation & bug reporting acknowledgment

Yes, I read it

### Describe your problem

Hello. I have built a glpi agent through a proxy application.
Over time, the agent has a message in the logs [info] [proxy server plugin] request rate limitation applied for remote 127.0.0.1
glpi agent v1.4
How to deal with it?

**Comment by g-bougard** [comment]
Hi @Provotorov1592 

you can tune `maxrate` & `maxrate_period` values in your Proxy plugin configuration as explained in the default configuration file: https://github.com/glpi-project/glpi-agent/blob/develop/etc/proxy-server-plugin.cfg

**Comment by rusmongoose** [comment]
Thanks
